cmd/oras/login.go

```diff
@@ -76,30 +75,37 @@ Example - Log in with username and password in an interactive terminal and no TL
 func runLogin(opts loginOptions) (err error) {
 	ctx, _ := trace.NewLogger(opts.Debug, opts.Verbose)
 
-	// prompt for credential
 	if opts.Password == "" {
+		// prompt for credential
+		reader := bufio.NewReader(os.Stdin)
 		if opts.Username == "" {
-			// prompt for username
-			username, err := readLine("Username: ", false)
+			fmt.Print("username: ")
```

Comment:
We should keep it `Username` instead of `username`. Same applies to other parts.